relay-server/src/utils/profile.rs

```diff
@@ -287,6 +287,110 @@ pub fn parse_typescript_profile(item: &mut Item) -> Result<(), ProfileError> {
     Ok(())
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+struct RustProfile {
+    duration_ns: u128,
+    platform: String,
+    architecture: String,
+    // string representation of Uuid v4
+    trace_id: String,
+    transaction_name: String,
+    // string representation of Uuid v4
+    transaction_id: String,
+    // string representation of Uuid v4
+    profile_id: String,
+    sampled_profile: RustSampledProfile,
+    device_os_name: String,
+    device_os_version: String,
+    version_name: String,
+    version_code: String,
+    debug_meta: RustDebugMeta,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct RustDebugMeta {
```

Comment:
nit: I think we have a convention of declaring types before they are used, so in this case `RustDebugMeta` before `RustProfile`. Maybe @jan-auer can confirm whether we still maintain that convention.